packages/TesterApp/android/gradle.properties

```diff
@@ -34,8 +39,15 @@ reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
 # your application. You should enable this flag either if you want
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
-newArchEnabled=false
```

Comment:
any reason why is it `false`?

Reply:
we can enable that, the only thing Re.Pack doesn't support right now is bridgeless mode - from what I remember, on iOS new arch is currently enabled